wemake_python_styleguide/visitors/ast/blocks.py

```diff
@@ -73,6 +73,8 @@ class BlockVariableVisitor(base.BaseNodeVisitor):
 
     """
 
+    _ast_func = (ast.FunctionDef, ast.AsyncFunctionDef)
```

Comment:
we already have this defined: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/compat/aliases.py#L19